plaso/parsers/jsonl_plugins/gcp_log.py

```diff
@@ -129,6 +133,17 @@ def _ParseProtoPayload(self, json_dict, event_data):
     if method_name and not event_data.event_subtype:
       event_data.event_subtype = method_name
 
+    status = self._GetJSONValue(proto_payload, 'status')
+    if status:
+      event_data.status_message = self._GetJSONValue(status,'message')
```

Comment:
style nit: here and for similar ones below, add a space between arguments i.e. `self._GetJSONValue(status,'message')` -> `self._GetJSONValue(status, 'message')`